test/mainnet-fork/Uniswapv4.t.sol

```diff
@@ -839,6 +863,154 @@ contract MainnetControllerUniswapV4Tests is ForkTestBase {
         assertEq(result.amount1Received, 159.209952e6);
     }
 
+    /**********************************************************************************************/
+    /*** swapUniswapV4 Tests                                                                   ***/
+    /**********************************************************************************************/
+
+    function test_swapUniswapV4_reentrancy() external {
+        _setControllerEntered();
+        vm.expectRevert(ReentrancyGuard.ReentrancyGuardReentrantCall.selector);
+        mainnetController.swapUniswapV4(bytes32(0), address(0), 0, 0);
+    }
+
+    function test_swapUniswapV4_revertsForNonRelayer() external {
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                IAccessControl.AccessControlUnauthorizedAccount.selector,
+                _unauthorized,
+                mainnetController.RELAYER()
+            )
+        );
+
+        vm.prank(_unauthorized);
+        mainnetController.swapUniswapV4(bytes32(0), address(0), 0, 0);
+    }
+
+    function test_swapUniswapV4_revertsWhenMaxSlippageNotSet() external {
+        vm.prank(relayer);
+        vm.expectRevert("MC/max-slippage-not-set");
+        mainnetController.swapUniswapV4(_POOL_ID, address(0), 0, 0);
+    }
+
+    function test_swapUniswapV4_revertsWhenAmountOutMinTooLow() external {
```

Comment:
Forgot to mention this in the last PR, all of the boundary style tests in this file should have the name to indicate it `test_swapUniswapV4_revertsWhenAmountOutMinTooLowBoundary`